.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,65 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+	"name": "Curriculum",
+	"dockerComposeFile": "../docker-compose.yml",
+	"service": "curriculum",
+	"workspaceFolder": "/app",
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Uncomment the next line to run commands after the container is created.
+	// "postCreateCommand": "cat /etc/os-release",
+
+	"mounts": [
+		"source=vscode_extensions,target=/root/.vscode-server/extensions,type=volume"
+	],
+	// Configure tool-specific properties.
+	// "customizations": {},
+	"customizations": {
```

Comment:
I don't know much about VS Code's configurations. Is this going to install all these extensions when someone opens the dev container?

If so, could you please restrict it to the bare minimum required to get going, and if there are some optional but nice ones, put them in the [extensions.json](https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions) as recommendations.

I would be interested to know the difference between using the rubocop gem over the standardrb gem too https://github.com/standardrb/vscode-standard-ruby

Reply:
Yeah good point, I just copied what we had in TC, but gone through and had a clear up, and moved several into recommendation as suggested.

I have not put vscode-standard-ruby in, as will do a seperate branch to do the updates, and will add it in once I have standardrb in place